app/services/caregivers_assistance_claims_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class CaregiversAssistanceClaimsService
+  def submit_claim!(user_context, claim_data)
+    claim = SavedClaim::CaregiversAssistanceClaim.new(claim_data)
+
+    unless claim.valid?
```

Comment:
You might consider moving this to the controller -- something like the pattern in `EducationBenefitsClaimsController` -- handling the client-related issues (valid data, corresponding StatsD work) in the controller and the more server-oriented work here.